ftml/test/anchor2-end.json

```diff
@@ -0,0 +1,42 @@
+{
+    "input": "[[a_ href=\"/some-page\"]]My link[[/a_]]",
```

Comment:
I recall agreeing that we wouldn't implement this (#9)

Reply:
PR #9 was about adding it to the Wikidot regex, discussing about its unnecessary complexity. Also the issue [WJ-147](https://scuttle.atlassian.net/browse/WJ-147) ("Multiple ways of typing something") was created addressing a similar issue.

This test only verifies existing behavior. Block rules permit any valid name to terminate the ending block. If we implement the first suggestion from [your reply here](https://github.com/scpwiki/wikijump/pull/123#discussion_r589998768) then we could definitely make it so that underscores like this are not accepted.

Reply:
Fair enough. If it's too much effort to implement, I'd be okay with this provided that it's not documented anywhere. I'm never certain what the best balance of flexibility vs strictness is. I'd prefer to err on the strict side but this is pretty harmless.

Reply:
Okay makes sense. Yeah it was less of an intentional addition and more about permissiveness. I think I'll keep the behavior but only have the one test for it, and not really document it.